src/payment/checkout/payment-form/StripePaymentForm.jsx

```diff
@@ -79,7 +52,7 @@ function StripePaymentForm({
       // Make sure to disable form submission until Stripe.js has loaded.
       return;
     }
-
+    setMessage('');
```

Comment:
Do we need this here? The use of the message state has been removed on this PR, so we should also remove L139, we don't want to display just an empty string imo.

Reply:
And we wouldn't need L24 useState for this message either

Reply:
This is to clear any old message at the start of a submit(ie a user enters an invalid card the message will tell them their card info is incorrect then when they hit submit again the message will clear and either stay cleared or be populated by any new error message). The message state still exists and is currently updated in StripePaymentMethodHandler to show any errors from the updatePaymentIntent.  The useEffect that was removed was a watcher on the stripe state when we were confirming Payment directly to Stripe but now with the custom actions beta it doesnt make sense to have that anymore.